en/services/opendroneid.md

```diff
@@ -46,27 +51,53 @@ Basic ID | Basic ID | [OPEN_DRONE_ID_BASIC_ID](../messages/common.md#OPEN_DRONE_
+- | - | [OPEN_DRONE_ID_ARM_STATUS](../messages/common.md#OPEN_DRONE_ID_ARM_STATUS) | Broadcast once per second from the RID transmitter/receiver components, to indicate their state. Similar to [Heartbeat](#heartbeat), but with additional text information in the error field when they are not ready to arm.
+- | - | [OPEN_DRONE_ID_SYSTEM_UPDATE](../messages/common.md#OPEN_DRONE_ID_SYSTEM_UPDATE) | A subset of the [OPEN_DRONE_ID_SYSTEM](#OPEN_DRONE_ID_SYSTEM) message, containing only fields that must be updated regularly by the GCS. Can optionally be used instead of the full `OPEN_DRONE_ID_SYSTEM` message, to reduce the traffic on the control link.
+
 
-> **Note** The raw byte layout of the MAVLink messages is not exactly the same as what a drone ID Bluetooth/Wi-Fi transmitter will transmit over the air.
+> **Note** The raw byte layout of the MAVLink messages is not exactly the same as what a RID Bluetooth/Wi-Fi transmitter component will broadcast over the air.
   Slight compression is applied.
   Example code for this conversion can be found in the project: [Open Drone ID Core C Library](https://github.com/opendroneid/opendroneid-core-c).
 
-The [Open Drone ID Core C Library](https://github.com/opendroneid/opendroneid-core-c) contains code for decoding the MAVLink messages and "compressing" the data into data structures for transmission over Bluetooth or Wi-Fi (or vice-versa).
-
-The standards requires that the Location message is broadcast/published at least once per second.
-The rest of the messages must be broadcast/published once per 3 seconds (requirements from local legislation might be different.
-Please see [here](https://github.com/opendroneid/opendroneid-core-c#comparison)).
-Not all message types are mandatory to broadcast.
+The [Open Drone ID Core C Library](https://github.com/opendroneid/opendroneid-core-c) contains code for decoding the MAVLink messages and "compressing" the data into data structures for broadcast over Bluetooth or Wi-Fi (or vice-versa for reception).
 
 The standards do not impose any requirements for a drone to be capable of receiving drone ID messages, nor any requirements for reacting to their content (requirements from local legislation might be different).
 
-An example Android receiver implementation for broadcast drone ID messages is available here: [OpenDroneID Android receiver application](https://github.com/opendroneid/receiver-android).
+An example Android receiver implementation for broadcast drone ID messages is [available here](https://play.google.com/store/apps/details?id=org.opendroneid.android_osm) ([source codes](https://github.com/opendroneid/receiver-android)).
 
 Code related to (Internet) Network Remote ID can be found in the [InterUSS Project](https://github.com/interuss) and https://github.com/uastech/standards (Unofficial reference for UAS-related APIs).
 
+## Message update rates {#update_rates}
```

Comment:
This is all about the standards and what might change. As a reader I'm more interested in clear instructions. 

So 
1. Point out the drivers
2. List the expected rates for known standards based on what PX4 and ArduPilot are actually setting for each case
4. A note that these are all guidelines and that systems will still need to be validated.

This is just my thinking. You could argue that this approach discourages users actually checking the standards, which they must do. But I was hoping we could give setting that will nominally "do the job" and that might not have to be changed.

--- 

MAVLink message streaming rates are largely driven by the requirements for compliance to various [standards](#specifications) (including ASTM F3411 and ASD-STAN prEN 4709-00). Generally rates should be tuned to the lowest rate that guarantees meeting the standards in the region in which the system is to be used.

For compliance with ASTM F3411 and ASD-STAN prEN 4709-00 commonly used rates are:
- OPEN_DRONE_ID_SYSTEM_UPDATE : 2 Hz  [this is  a made-up list]
- OPEN_DRONE_ID_SYSTEM_UPDATE: 0.1 Hz (1 Hz if OPEN_DRONE_ID_SYSTEM_UPDATE is not used)
- ETC
- 
 In addition  the following rates are recommended, but not required for compliance:
- OPEN_DRONE_ID_ARM_STATUS: 1Hz.

Note all messages are mandatory. Implementers are expected to check relevant standards blah blah.

Reply:
I agree with you but right now I don't know what exact rates will be used in Ardupilot and PX4. We will need to do this update later on, once the implementation work settles down.